webpack.config.js

```diff
@@ -16,9 +16,13 @@ module.exports = {
   plugins: [
     new HTMLWebpackPlugin({
       hash: true,
-      title: 'Genshin Impact Wish Sim',
+      title: 'Genshin Impact Wish Simulator',
       template: `${__dirname}/src/index.html`,
       filename: `${__dirname}/dist/index.html`,
+      metaName: 'Genshin Impact Wish Simulator',
+      metaDesc: 'A simulator to let your wish come true. Test your luck here',
+      metaUrl: 'https://uzair-ashraf.github.io/genshin-impact-wish-simulator',
+      metaImage: `${__dirname}/src/assets/images/banners/sparkling-steps.png`
```

Comment:
Url is different now, and we should use the banner for the current character event wish

Reply:
sure, let me changed them. Sorry for long response.
By today, ganyu banner only out, do you have updated image? So I can help update on them?

Reply:
Ganyu banner is in repo